### PR TITLE
CURA-6517 check new net-printer visible in monitor

### PR DESCRIPTION
**Pull request by rburema**
**NOTE**: Please do the review of ticket CURA-6317 _first_ (see #8), as this code relies on that (and this branch included that one)!

### CODE REVIEW

File: suite_Cura/shared/scripts/PageObjects/CommonPage.py [hunk 1]
```diff
@@ -151,12 +151,15 @@ def click(obj, time_out=15000):
...
+    def findObjectWithText(self, object, value, property='text', lang=None, exact_match=False):
         if lang is not None:
             value = self.getTranslatedText(value, lang)
 
         obj = object.copy()
-        obj[property] = Wildcard("*" + value + "*")
+        if exact_match:
+            obj[property] = Wildcard("*" + value + "*")
+        else:
+            obj[property] = Wildcard(value)
```
Comment (Ghostkeeper) [inline suite_Cura/shared/scripts/PageObjects/CommonPage.py]: This `if` `else` seems to be the wrong way around.

File: suite_Cura/shared/scripts/PageObjects/CuraPage.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 from PageObjects.PerformancePage import Performance
 import names
 
+import builtins
```
Comment (Ghostkeeper) [inline suite_Cura/shared/scripts/PageObjects/CuraPage.py]: As I remember you telling me while ago, this was necessary to differentiate between `builtins.int` and the `int` function that Squish overrides. You might want to document that.

File: suite_Cura/shared/scripts/PageObjects/PrinterPage.py [hunk 3]
```diff
@@ -34,4 +36,25 @@ def navigateToPrinterPreferences(self):
...
+        return len(extruder_list)
+
+    def syncConfig(self):
+        if object.exists(names.btn_to_config):
+            self.click(names.btn_to_config)
+        elif not object.exists(names.mwi_printer_config_drop):
+            self.click(names.mwi_lst_extruders)
+
+            # Work-around the issue that the config.-button doesn't (always?) show up immediately.
+            check_time = time.time() + 20.0  # <- Don't attempt the workaround 'till the end of time.
```
Comment (Ghostkeeper) [inline suite_Cura/shared/scripts/PageObjects/PrinterPage.py]: [Reminder to myself to test this workaround well and whether it is necessary...]

File: suite_Cura/tst_Printer_Management/test.feature [hunk 4]
```diff
@@ -1,15 +1,42 @@
...
     When I navigate to menu Preferences and Configure Cura
     And I navigate to Printers in preferences
     And I select Add printer
     And I add a non-networked Ultimaker 2 Extended+ printer
     Then The printer overview contains a 'Ultimaker 2 Extended+' printer
     And I close the preferences
 
-  Scenario: Add networked printer from printer preferences
+  Scenario: Add detected networked printer from printer preferences
+	Given Cura is running
```
Comment (Ghostkeeper) [inline suite_Cura/tst_Printer_Management/test.feature]: Mixing tabs and spaces here.

File: suite_Cura/tst_Printer_Management/test.feature [hunk 5]
```diff
@@ -1,15 +1,42 @@
...
+    And I select Add printer
+    And I add a network printer with name 'Frankie'
+    Then The printer overview contains a 'Frankie' printer
+    And I close the preferences
+
+  Scenario: Check if networked printers show up in the monitor page
+	Given Cura is running
+	And I select the 'Frankie' printer and '0.1' profile
+	And I synchronize with the printers configuration
+	Then I observe 'Frankie' in the monitor page
```
Comment (Ghostkeeper) [inline suite_Cura/tst_Printer_Management/test.feature]: Mixing tabs and spaces here.

File: suite_Cura/shared/scripts/PageObjects/CommonPage.py (hunk 1 above)
Comment (rburema) [reply suite_Cura/shared/scripts/PageObjects/CommonPage.py]: Fixed! (The case where the inexact match is wrong isn't always reproducible, so it _seemed_ fixed this way...)

File: suite_Cura/shared/scripts/PageObjects/CuraPage.py (hunk 2 above)
Comment (rburema) [reply suite_Cura/shared/scripts/PageObjects/CuraPage.py]: I commented this now.

File: suite_Cura/tst_Printer_Management/test.feature (hunk 4 above)
Comment (rburema) [reply suite_Cura/tst_Printer_Management/test.feature]: Fixed!

File: suite_Cura/tst_Printer_Management/test.feature (hunk 5 above)
Comment (rburema) [reply suite_Cura/tst_Printer_Management/test.feature]: Fixed!

**Pull request merged**